.buildkite/steps/build-image.sh

```diff
@@ -9,6 +9,9 @@ else
     BUILDTYPE="latest"
 fi
 
-BUILDER_TAG=${BUILDER_TAG:-$BUILDTYPE}
+if [[ -z ${BUILDER_TAG} ]]; then 
```

Comment:
Try to use `-z ${BUILDER_TAG+x}` in condition. Then you won’t have to delete `set -u` in line 2. See: https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash